### PR TITLE
[TwigBundle] update the default Twig escape strategy

### PR DESCRIPTION
**Pull request by xabbuh**
reflect the changes made in symfony/symfony#20224

### CODE REVIEW

File: reference/configuration/twig.rst [hunk 1]
```diff
@@ -141,9 +141,9 @@ individually in the templates).
 
 If set to a string, the template contents are escaped using the strategy with
 that name. Allowed values are ``html``, ``js``, ``css``, ``url``, ``html_attr``
-and ``filename``. The default value is ``filename``. This strategy escapes
-contents according to the filename extension (e.g. it uses ``html`` for
-``*.html.twig`` templates and ``js`` for ``*.js.html`` templates).
+and ``name``. The default value is ``name``. This strategy escapes contents
+according to the filename extension (e.g. it uses ``html`` for ``*.html.twig``
```
Comment (javiereguiluz) [inline reference/configuration/twig.rst]: `according to the filename extension` -> `according to the template name extension` ?

**Review by javiereguiluz — APPROVED** [review]
👍

File: reference/configuration/twig.rst (hunk 1 above)
Comment (xabbuh) [inline reference/configuration/twig.rst]: done

**Comment by wouterj** [comment]
Thanks!

**Pull request merged**